Source/DafnyLanguageServer/Workspace/LanguageServerFilesystem.cs

```
@@ -29,13 +29,22 @@ private class Entry {
 
   private readonly ConcurrentDictionary<Uri, Entry> openFiles = new();
 
-  public void OpenDocument(TextDocumentItem document) {
+  public bool OpenDocument(TextDocumentItem document) {
     var uri = document.Uri.ToUri();
     if (openFiles.ContainsKey(uri)) {
       throw new InvalidOperationException($"Cannot open file {uri} because it is already open");
     }
 
+    string existingText = "";
+    try {
+      if (OnDiskFileSystem.Instance.Exists(uri)) {
+        existingText = OnDiskFileSystem.Instance.ReadFile(uri).ReadToEnd();
+      }
+    } catch (IOException) {
+
+    }
```

Is it safe to silence exceptions here, and if so can you explain in a comment?